scheduler/integration-test/package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "mock:adapter": "node src/mock.adapter.js",
     "pretest": "npm run mock:adapter &",
-    "test": "jest",
+    "test": "jest --runInBand",
```

Comment:
Necessary?

Reply:
Since the tests are heavily timing dependant (which is problematic anyways), I found that the `runInBand` flag imporoved the stability a little. So I would keep it for now, yes. Of course, it would be better if we found a more robust way to test the scheduler but I did not have an idea yet.

Reply:
I also prefer the `--runInBand` option. Several people reported that `--runInBand` helps in an environment with limited resources, like a CI environment.

See (https://github.com/facebook/jest/issues/1524#issuecomment-260246008)[https://github.com/facebook/jest/issues/1524#issuecomment-260246008]